conf/Makefile.chibios

```diff
@@ -246,9 +251,9 @@ TCSRC =
 TCPPSRC =
 
 # List ASM source files here
-ASMSRC = $(STARTUPASM) $(PORTASM) $(OSALASM)
+ASMXSRC = $(STARTUPASM) $(PORTASM) $(OSALASM)
```

Comment:
I just checked your commit - you are using ChibiOS `master` instead of `stable_16.1.x`
Are you sure you want to switch to the newest (and possibly) buggy ChibiOS? F7xx support seems to be in `stable_16.1.x` too...